app/src/main/java/com/mashup/mobalmobal/ui/createdonation/CreateDonationFragment.kt

```diff
@@ -0,0 +1,247 @@
+        imeActionId: Int,
+        viewIndex: Int,
+        nextViewLayout: View,
+        nextView: View?
+    ) {
+        setOnEditorActionListener { v, actionId, event ->
+            if (actionId == imeActionId && binding.createDonationProductImageView.visibility != View.VISIBLE) {
+                goDownAnimation(viewIndex, nextViewLayout, nextView)
+            }
+            false
+        }
+    }
+
+    private fun goDownAnimation(
+        viewIndex: Int,
+        nextViewLayout: View,
+        nextView: View?
+    ) {
+        ObjectAnimator.ofFloat(
+            binding.createDonationInputWrapperLayout,
+            "translationY",
+            binding.createDonationNameTextInputLayout.height.toFloat() * (viewIndex + 1)
+        ).apply {
+            duration = 300
+            start()
+        }.doOnEnd {
+            nextViewLayout.visibility = View.VISIBLE
+            nextView?.requestFocus()
+            (nextView as? EditText)?.showSoftInput()
+        }
+    }
+
+    private fun setDateTimePickerDialog() {
+        binding.createDonationStartDateInput.setOnClickListener {
+            showDatePickerDialog(START_PICKER_ID)
+        }
+        binding.createDonationDueDateInput.setOnClickListener {
+            showDatePickerDialog(END_PICKER_ID)
+        }
+    }
+
+    private fun showDatePickerDialog(viewId: Int) {
+        val c = Calendar.getInstance()
+        if (viewId == 2) {
+            c.add(Calendar.DAY_OF_MONTH, DAY_DIFF)
+        }
+        val year = c.get(Calendar.YEAR)
+        val month = c.get(Calendar.MONTH)
+        val day = c.get(Calendar.DAY_OF_MONTH)
+
+        val dateSetListener = DatePickerDialog.OnDateSetListener { view, year, month, dayOfMonth ->
+            showTimePickerDialog(viewId, year, month, dayOfMonth)
+        }
+
+        DatePickerDialog(requireActivity(), dateSetListener, year, month, day).show()
+    }
+
+    private fun showTimePickerDialog(viewId: Int, year: Int, month: Int, dayOfMonth: Int) {
+        val c = Calendar.getInstance()
+        if (viewId == 2) {
+            c.add(Calendar.DAY_OF_MONTH, DAY_DIFF)
+            c.add(Calendar.HOUR_OF_DAY, HOUR_DIFF)
+        }
+        val hour = c.get(Calendar.HOUR_OF_DAY)
+        val minute = c.get(Calendar.MINUTE)
+
+        val timeSetListener = TimePickerDialog.OnTimeSetListener { view, hourOfDay, minute ->
+            val calendar = Calendar.getInstance().also {
+                it.set(Calendar.YEAR, year)
+                it.set(Calendar.MONTH, month)
+                it.set(Calendar.DAY_OF_MONTH, dayOfMonth)
+                it.set(Calendar.HOUR_OF_DAY, hourOfDay)
+                it.set(Calendar.MINUTE, minute)
+            }
+            val formattedDateTime = getFormattedDateTime(calendar.timeInMillis)
+
+            if (viewId == 1) {
+                createDonationViewModel.setCreateDonationStartDate(calendar.timeInMillis)
+                binding.createDonationStartDateInput.setText(formattedDateTime)
```

Comment:
```suggestion
                binding.createDonationStartDateInput.text = formattedDateTime
```
setText 그대로 쓰는건 java style 입니다

Reply:
createDonationStartDateTimeInput가 EditText 라서 setText를 사용했습니다:)